editor/editor_help.cpp

```diff
@@ -335,6 +352,14 @@ void EditorHelp::_add_method(const DocData::MethodDoc &p_method, bool p_overview
 		class_desc->pop();
 	}
 
+	if (p_method.is_deprecated) {
+		DEPRECATED_DOC_TAG
+	}
+
+	if (p_method.is_experimental) {
+		EXPERIMENTAL_DOC_TAG
+	}
```

Comment:
It's not needed technically but macros look cleaner if they look like actual statements IMO.
```suggestion
	if (p_method.is_deprecated) {
		DEPRECATED_DOC_TAG;
	}

	if (p_method.is_experimental) {
		EXPERIMENTAL_DOC_TAG;
	}
```